.husky/pre-commit

```diff
@@ -1,4 +0,0 @@
-#!/bin/sh
-. "$(dirname "$0")/_/husky.sh"
-
-npm run lint && npm run format && git add -A .
```

Comment:
the `git add -A` command will unintentionally add all unstaged files to a commit without notice